front/src/modules/pathfinding/components/IncompatibleConstraints/IncompatibleConstraintsItem.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { type FC } from 'react';
+
+import { Location } from '@osrd-project/ui-icons';
+import cx from 'classnames';
+import { isNil } from 'lodash';
+import { useTranslation } from 'react-i18next';
+
+import { distanceToHumanReadable } from 'utils/strings';
+
+import type { IncompatibleConstraintItemEnhanced } from './type';
+import { getIcon } from './utils';
+
+interface IncompatibleConstraintItemProps {
+  data: IncompatibleConstraintItemEnhanced;
+  isHovered?: boolean;
+  isSelected?: boolean;
+  onEnter: () => void;
+  onLeave: () => void;
+  onClick: () => void;
+  gotoMap: () => void;
+}
+
+const IncompatibleConstraintItem: FC<IncompatibleConstraintItemProps> = ({
+  data,
+  isHovered,
+  isSelected,
+  onEnter,
+  onLeave,
+  onClick,
+  gotoMap,
+}) => {
+  const { t } = useTranslation(['operationalStudies/manageTrainSchedule']);
+  const Icon = getIcon(data.type);
+  let bgClass = 'bg-light';
+  if (isSelected === true) bgClass = 'bg-success text-white';
+  if (isHovered === true) bgClass = 'bg-info text-white';
+
+  return (
+    <div
+      className={cx(
+        'incompatible-constraints-item d-flex justify-content-around align-items-center my-2 rounded py-1 ',
```

Comment:
As discussed, even with the align-center, the text feels like it's more on top (probably to handle the "long letters"). Is this change ok for you ? It's also close to the behavior @thibautsailly tries to implement in the new osrd ui.

![Capture d’écran 2024-08-02 à 16 45 36](https://github.com/user-attachments/assets/23f7dbf6-bb21-4de7-873d-2c19c3562131)

```suggestion
        'incompatible-constraints-item d-flex justify-content-around align-items-center my-2 rounded pt-1 ',
```

Reply:
If you change the `py-1` by `pt-1`, the whole inner item will be changed, and not only the text.
I  did some changes, and it's better now